root.c

```diff
@@ -495,6 +495,31 @@ luaA_root_wallpaper(lua_State *L)
     return 1;
 }
 
+
+/** Get the content of the root window as a cairo surface
+ *
+ * @return A cairo surface
+ * @staticfct content
```

Comment:
I think it should be documented as a module level property (aka field). But it seems other similar fields are documented as object properties, tho. So I think we should also use :

```suggestion
 * @property content
 * @tparam surface A cairo surface with the root window content (aka the whole surface from every screens).
 * @see gears.surface
```

Reply:
Is it a property? It appeared to me that root is a library that only has functions? I'll press accept if this is what is wanted though.
https://awesomewm.org/doc/api/libraries/root.html